Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue8797.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Xamarin.Forms.CustomAttributes;
+
+namespace Xamarin.Forms.Controls.Issues
+{
+	[Issue(IssueTracker.Github, 8797, "[Bug] Word wrapped Label not measured correctly",
+		PlatformAffected.Android | PlatformAffected.iOS)]
+	public class Issue8797 : TestContentPage
+	{
+		protected override void Init()
+		{
+			var scrollView = new ScrollView();
+			var layout = new StackLayout { Margin = new Thickness(5, 80, 5, 0) };
+
+			var instructions = new Label { Text = "The text visible in the Grid should end with 'finding a good way to spend it'. If that text is cut off, this test has failed." };
```

Comment:
This is working fine!
<img width="504" alt="Captura de pantalla 2020-07-08 a las 12 58 02" src="https://user-images.githubusercontent.com/6755973/86910930-bea4c000-c11a-11ea-8977-d05b6bfb420a.png">